HW02_B_Mimic_starter.Rmd

```diff
@@ -75,7 +78,12 @@ set.seed(42)
 Now using the mpg dataset and the corvette dataset, make this graph:
 
 ```{r graoh 3 code}
-
+ggplot(mpg,mapping = aes(x = displ, y = hwy)) + 
```

Comment:
This one looks amazing! You made the corvettes a different color in a very elegant way that I could not figure out, so I'm impressed. The only differences between your graph and the original are the gridlines and the x-axis number labels. Note that the original had more gridlines and that the x-axis number labels were from 1 to 8, going up by an increment of 1. Yours is going up by an increment of 2. You could make those changes by adding this: scale_x_continuous(breaks = c(1:8), limit = c(1,8))